encoding/descaling.md

```diff
@@ -1,477 +1,365 @@
+merged = core.std.MaskedMerge(rescaled, planes[0], credit_mask)
 
-```Py
-descaled = kgf.inverse_scale(src, height=878, kernel='bicubic', b=0, c=1/2, mask_detail=True)
+merged = core.std.ShufflePlanes([merged, clip], [0, 1, 2], vs.YUV)
 ```
 
-We can make use of the mask
-that `inverse_scale` uses internally
-as well.
+![](images/descale/descaled_symphoxv_credits_frame_non-masked.png)
 
-```Py
-descaled = kgf.inverse_scale(src, height=874, kernel='bicubic', b=0, c=1/2)
-descaled_mask = kgf._generate_descale_mask(vsutil.get_y(core.resize.Spline36(src, descaled.width, descaled.height)), vsutil.get_y(descaled), kernel='bicubic', b=0, c=1/2)
-```
+![A mask to catch the credits in Symphogear XV's OP](images/descale/descaled_symphoxv_credits_mask.png)
+![Merge the original credits with a re-upscaled frame](images/descale/descaled_symphoxv_credits_frame_masked.png)
 
-![Kaguya-sama: Love Is War — OP (credits mask)](images/descale_credits_mask.png)
-
-![Kaguya-sama: Love Is War — OP (descaled)](images/descale_credits.png)
-
-Note that if you see the mask
-catching a lot of other stuff,
-you might want to consider *not* descaling
-that particular frame,
-or trying a different kernel/values.
-Chances are that you're either
-using the wrong kernel
-or that the frames you're looking at are native 1080p.
-
-![Manaria Friends — 01 (end card)](images/descale_native1080.png)
-
-![(Don't descale this)](images/descale_dontdescalethis.png)
-
-
-## Dealing with bad descaling
-
-There are various things you can do
-to deal with scenes that have issues
-even after descaling.
-`Eedi3` stands out in particular
-as a fantastic AA filter
-that really nails down bad lineart
-caused by bad descaling.
-It's at best a "cheat code",
-however.
-While it might fix up some issues,
-it won't fix everything.
-It's also incredibly slow,
-so you might want to
-[use it on just a couple of frames at a time](scenefiltering.md)
-rather than over the entire clip.
-
-Other than Eedi3,
-usually the results of bad descaling
-are so destructive that
-there isn't much you can do.
-If you have an encode that's
-badly descaled,
-you're better off
-finding a different one.
-If you've got bad results
-after descaling yourself,
-try a different kernel or values.
-Alternatively,
-try not descaling at all.
-
-At the end of the day,
-as mentioned in the introduction,
-you can't descale everything perfectly.
-Sometimes it's better to think of it
-as a magical anti-ringing/haloing/aliasing filter
-rather than a scaler.
-
-For example,
-here it was used
-specifically to fix up
-some bad aliasing
-in the source.
-
-![Akanesasu Shoujo — 01 (src)](images/descale_akanesasu_src.png)
-
-![Akanesasu Shoujo — 01 (rescaled)](images/descale_akanesasu_rescaled.png)
+[Here][rescaled] is a link to a comparison.
+
+This can also be done when descaling
+to a resolution that you plan to stick with.
+Since the credits will still be messed up,
+you will need to replace them with a downscaled clip.
```

Comment:
The differentiation here is that you are going to *downscale* the credits while *descaling* the rest of the frame, right? I think better wording would be as follows:
```suggestion
Since there is no need to *descale* native 1080p elements,
you will need to *downscale* the credits instead.
```